### ISSUE TITLE
Nanobench fails to update

### ISSUE THREAD

**Issue by kmhettne** [peta-pico/nanobench#63]
My nanobench installation is about two years old, I think it might even be the first nanobench. If I run ./update it fails to update. Should I reinstall?

I get the following error message:

May 19, 2022 1:13:42 PM org.apache.coyote.AbstractProtocol init
INFO: Initializing ProtocolHandler ["http-bio-8080"]
May 19, 2022 1:13:42 PM org.apache.catalina.core.StandardService startInternal
INFO: Starting service Tomcat
May 19, 2022 1:13:42 PM org.apache.catalina.core.StandardEngine startInternal
INFO: Starting Servlet Engine: Apache Tomcat/7.0.37
May 19, 2022 1:13:42 PM org.apache.catalina.loader.WebappLoader setClassPath
INFO: Unknown loader jdk.internal.loader.ClassLoaders$AppClassLoader@55054057 class jdk.internal.loader.ClassLoaders$AppClassLoader

----------------------------------------
         Updating Nanobench...
----------------------------------------
 Nanobench is being updated. This might
 take a minute.
----------------------------------------

WARN  - ResponseProcessCookies     - Invalid cookie header: "Set-Cookie: _octo=GH1.1.6921694.1652958823; Path=/; Domain=github.com; Expires=Fri, 19 May 2023 11:13:43 GMT; Secure; SameSite=Lax". Invalid 'expires' attribute: Fri, 19 May 2023 11:13:43 GMT
WARN  - ResponseProcessCookies     - Invalid cookie header: "Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 19 May 2023 11:13:43 GMT; HttpOnly; Secure; SameSite=Lax". Invalid 'expires' attribute: Fri, 19 May 2023 11:13:43 GMT
WARN  - ResponseProcessCookies     - Invalid cookie header: "Set-Cookie: _octo=GH1.1.1234507147.1652958824; Path=/; Domain=github.com; Expires=Fri, 19 May 2023 11:13:44 GMT; Secure; SameSite=Lax". Invalid 'expires' attribute: Fri, 19 May 2023 11:13:44 GMT
WARN  - ResponseProcessCookies     - Invalid cookie header: "Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 19 May 2023 11:13:44 GMT; HttpOnly; Secure; SameSite=Lax". Invalid 'expires' attribute: Fri, 19 May 2023 11:13:44 GMT
Failed to update Nanobench

**Comment by tkuhn** [comment]
Yes, reinstalling should solve this. There was a bug with updating in older versions.

You can at any time delete the nanobench directory that you get from extracting the downloaded zip file. Only make sure to keep the `.nanopub` directory that is in your home folder. All configuration, including your private key, is stored there.

I hope reinstalling resolves the issue. Let me know if not.

**Comment by kmhettne** [comment]
Reinstalling worked, thank you for the fast reply.

**Comment by tkuhn** [comment]
Great to hear!
